agenta-web/src/components/EvaluationTable/AICritiqueEvaluationTable.tsx

```diff
@@ -231,6 +261,30 @@ Answer ONLY with one of the given grading or evaluation options.
         }
     }
 
+    const handleExportClick = () => {
```

Comment:
Can we please create a helper function for this under `@/lib/helpers/evaluate.ts` to avoid the repetition of code?

A function with signature like:

```javascript
const exportEvaluationData = (evaluation, rows) => {
   ...
   // function body
}
```